### ISSUE TITLE
ERROR: bundling failed: Error: Unable to resolve module `brorand-shim`

### ISSUE THREAD

**Issue by aerielcruz** [uport-project/react-native-uport-connect#4]
**Error after importing `import configureUportConnect from 'react-native-uport-connect';`**
```
error: bundling failed: Error: Unable to resolve module `brorand-shim` from 
`/Users/username/Project/node_modules/elliptic/lib/elliptic.js`: 
Module `brorand-shim` does not exist in the Haste module map
```

**Comment by aerielcruz** [comment]
## Solved!
The error occurred after installing `@babel/preset-react` which is a possible [solution](https://github.com/babel/babel/issues/6808#issuecomment-343792786) to the `Plugin/Preset files are not allowed to export objects, only functions.` error which lead to `"brorand": "brorand-shim",` error that was added in babel-preset.
**Before (ERROR)**
```
module.exports = () => {
	return {
		sourceMaps: true,
		plugins: [
			["rewrite-require", {
				"aliases": {
                                        "brorand": "brorand-shim",
					"crypto": "crypto-browserify",
					"stream": "stream-browserify",
					"process": "process/browser.js",
					"zlib": "zlib-browserify",
					"vm": "vm-browserify"
				}
			}]
			,
			[
				"module-resolver",
				{
					alias: {
						"randombytes": require.resolve("./randombytes.js"),
					},
				},
			],
		],
	}
};
```
**After (FIXED)**
Just removed `"brorand": "brorand-shim",`
```
module.exports = () => {
	return {
		sourceMaps: true,
		plugins: [
			["rewrite-require", {
				"aliases": {
					"crypto": "crypto-browserify",
					"stream": "stream-browserify",
					"process": "process/browser.js",
					"zlib": "zlib-browserify",
					"vm": "vm-browserify"
				}
			}]
			,
			[
				"module-resolver",
				{
					alias: {
						"randombytes": require.resolve("./randombytes.js"),
					},
				},
			],
		],
	}
};
```

**Comment by phoniks** [comment]
This helped me.  Thanks!

**Comment by phoniks** [comment]
Alright looks like I spoke too soon.  I'm running into this same problem but the above solution helped me find the problem, but the solution above didn't work.  Digging a little deeper, I realized that the error was due to having run `npm i` on a project that was using yarn.  Deleting the package-lock.json file and running `yarn cache clean` solved it for me.  If you're running into a similar issue try that.
